atomicfu-gradle-plugin/src/main/kotlin/kotlinx/atomicfu/plugin/gradle/AtomicFUGradlePlugin.kt

```diff
@@ -193,8 +193,8 @@ fun Project.withPluginWhenEvaluatedDependencies(plugin: String, fn: Project.(ver
 }
 
 fun Project.withKotlinTargets(fn: (KotlinTarget) -> Unit) {
-    extensions.findByType(KotlinProjectExtension::class.java)?.let { kotlinExtension ->
-        val targetsExtension = (kotlinExtension as? ExtensionAware)?.extensions?.findByName("targets")
+    extensions.findByType(KotlinTargetsContainer::class.java)?.let { kotlinExtension ->
+        val targetsExtension = kotlinExtension.targets
```

Comment:
Could be directly assigned to `val targets` plus remove unnecessary case

Reply:
Simplified, thanks